src/mumble/IPCUtils.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "IPCUtils.h"
+extern "C" {
+#include "../../overlay/ipc_utils.h"
+}
+
+#include <QDir>
+#include <QProcessEnvironment>
+#include <QString>
+
+namespace Mumble {
+const std::string getRuntimePath(void) {
+	char *c_mumbleRuntimePath = getRuntimePath__();
```

Comment:
You have to delete the allocated buffer once you are done with it

Reply:
You still have to delete the source buffer as converting a C string into a `std::string` will copy the buffer, not take ownership of it.

Reply:
You could also use `std::unique_ptr` as a wrapper.

Reply:
I think it's not possible to create a `std::string` directly from a `char*` array without copying. I thus doubt using `std::unique_ptr` would help, unless I misunderstood your point :\